src/test/kotlin/org/jetbrains/research/transformations/util/PythonTransformationsTest.kt

```diff
@@ -0,0 +1,48 @@
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.ParserSetup
+import org.jetbrains.research.transformations.util.Util
+import org.jetbrains.research.transformations.util.toXMLWithoutRoot
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.TestInstance
+import java.io.File
+import java.util.logging.Logger
+import kotlin.reflect.KFunction
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+open class PythonTransformationsTest {
+    protected val LOG = Logger.getLogger(javaClass.name)
+
+    protected val XMLTreeFileName = "${Util.getTmpPath()}/test.xml"
+
+    companion object {
+        fun getResourcePath(resourceName: String, cls: KFunction<PythonTransformationsTest>): String {
+            return cls.javaClass.getResource(resourceName).path
+        }
+    }
+
+    @BeforeAll
+    internal fun beforeAll() {
+        LOG.info("beforeAll called")
+        ParserSetup.checkSetup()
+    }
+
+    protected fun getSourcePythonCode(ctx: TreeContext, XMLDstPath: String): String {
```

Comment:
getSourcePythonCode -> getPythonSourceCode, see [this guide](https://dictionary.cambridge.org/ru/%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/%D0%B1%D1%80%D0%B8%D1%82%D0%B0%D0%BD%D1%81%D0%BA%D0%B0%D1%8F-%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/adjectives-order)